### PR TITLE
http_util.go: fix word in error message

### PR DESCRIPTION
**Pull request by ethan-daocloud**
Signed-off-by: Guangming Wang <guangming.wang@daocloud.io>

### Ⅰ. Describe what this PR did
fix typos

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)

yes.

### CODE REVIEW

File: pkg/httputils/http_util.go [hunk 1]
```diff
@@ -310,7 +310,7 @@ func GetRangeSE(rangeHTTPHeader string, length int64) ([]*RangeStruct, error) {
 	if strings.ContainsAny(rangeHTTPHeader, "=") {
 		rangeSlice := strings.Split(rangeHTTPHeader, "=")
 		if len(rangeSlice) != 2 {
-			return nil, errors.Wrapf(errortypes.ErrInvalidValue, "invaild range: %s, should be like bytes=0-1023", rangeStr)
```
Comment (allencloud) [inline pkg/httputils/http_util.go]: I found that there are two places in this file, which has the same incorrect spelling.
Could you help to fix that in this single pull request? @ethan-daocloud

Comment (ethan-daocloud) [reply pkg/httputils/http_util.go]: sure, I will find and fix that.

Comment (ethan-daocloud) [reply pkg/httputils/http_util.go]: @allencloud 
And I check this file thoroughly, find one more , please confirm :)

**Comment by allencloud** [comment]
LGTM

**Pull request merged**